PRIMER.md

```diff
@@ -363,9 +363,9 @@ Results in:
 
 Polymer supports extending custom element prototypes with shared code modules called "behaviors".
 
-A behavior is simply an object that looks very similar to a typical Polymer prototype.  It may define lifecycle callbacks, `properties`, `hostAttributes`, or other features described later in this document like `observers` and `listeners`.  To add a behavior to a Polymer element definition, include it in a `behaviors` array on the prototype.
+A behavior is simply an object that looks very similar to a typical Polymer prototype.  It may define lifecycle callbacks, `properties`, `hostAttributes`, or other features described later in this document like `observers` and `listeners`.  To add a behavior to a Polymer element definition, include it in a `behaviors` array on the prototype.  If multiple behaviors for the same element define the same function, only the last behavior's function is used.
```

Comment:
Does that mean that if the element has a properties object, the behavior can't define any properties? That doesn't seem right. Or that if the host defines a specific property, the behavior's definition of that same property is ignored?